src/neuroglancer/viewer.ts

```diff
@@ -742,27 +790,18 @@ export class Viewer extends RefCounted implements ViewerState {
     new WhatsNewDialog(this);
   }
 
-  loadFromJsonUrl() {
-    var urlParams = new URLSearchParams(window.location.search);
-    if (urlParams.has('json_url')) {
-      let json_url = urlParams.get('json_url')!;
-      history.replaceState(null, '', removeParameterFromUrl(window.location.href, 'json_url'));
-      StatusMessage.forPromise(
-          cancellableFetchOk(json_url, {}, responseJson).then(response => {
-            this.state.restoreState(response);
-          }),
-          {
-            initialMessage: `Retrieving state from json_url: ${json_url}.`,
-            delay: true,
-            errorPrefix: `Error retrieving state: `,
-          });
-    }
-  }
-
   showReportDialog(image: string) {
     new UserReportDialog(this, image);
   }
 
+  showSaveDialog(get?: UrlType, jsonString?: string) {
```

Comment:
Copied from `SaveDialog`: `get` should be a bit more descriptive, e.g. `getUrlType`